### DISCUSSION TITLE
Enable sponsors?

### DISCUSSION THREAD

**Discussion by audrow** [lumeland/lume#154] [General]
Hey, do you have plans to enable sponsorship for lume? I like where the project is going and would be happy to provide some support.

**Comment by oscarotero** [comment]
Hi, @audrow That would be great. Thank for your support.
I'm going to enable it and let you know when it's ready.

**Comment by oscarotero** [comment]
Hi again.
I just created an Open Collective page for sponsorship. Take a look at: https://opencollective.com/lume
I will include the sponsors logos/avatars in the Lume website.

With the first financial contributions I will, finally, purchase a domain for Lume website :)
Thanks!

**Reply by audrow** [reply]
Great, I'm contributing monthly. Thanks for setting it up!

**Reply by oscarotero** [reply]
Thank you for the support!

**Comment by aiotter** [comment]
Don't you have any plan to enable GitHub Sponsors? I prefer it to Open Collective.
I begun supporting Lume, but my budget is small. Open Collective's fee (for Stripe) costs too much for small amount of recursive payment. GitHub does not charge any fee to user accounts. That's nicer.

**Reply by oscarotero** [reply]
Hi @aiotter 

Thanks for support Lume. I tried to enable GitHub Sponsors, but I couldn't. The lumeland organization does not appear in my list of GitHub Sponsors eligible accounts. As you can see in this screenshots, other organizations that I'm belong to are in the list but not lumeland, so this is why I decided to use Open Collective.
Any idea of how to make it appears here? Maybe I missed something

<img width="632" alt="imaxe" src="https://user-images.githubusercontent.com/377873/147473655-61afb571-e5c7-4662-b7b8-c6668d0f87fd.png">

**Reply by aiotter** [reply]
Hmm... I checked my organizations and I can join the waitlist for all of my five organizations whose owner is me.
Spain is one of the available regions of GitHub Sponsors, so lumeland must be available. It seems weird.

You can contact GitHub from [here](https://support.github.com/contact). They may take time to reply, but they'll help you.

**Reply by oscarotero** [reply]
Thanks. I just contact them. As soon as I have a reply I'll let you know.

**Reply by oscarotero** [reply]
Hi again @aiotter. I can, finally, configure Lume to receive sponsors, but it seems that organizations sponsorship cannot be configured for individual persons, but companies. So cannot set up properly.
I'm going to include my own personal account for Github sponsors (https://github.com/sponsors/oscarotero) for now.

**Reply by aiotter** [reply]
@oscarotero Nice! Can you enable custom amount of donation? I want other options than only $1!

**Reply by oscarotero** [reply]
@aiotter Yes, sure! Thank you for your generosity!!
